Makefile

```diff
@@ -0,0 +1,29 @@
+# Copyright Contributors to the Open Cluster Management project
+
+
+default::
+	make help
+help:
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-25s\033[0m %s\n", $$1, $$2}'
```

Comment:
Added this to self-document the make targets.
![image](https://user-images.githubusercontent.com/4671325/138118945-a1196005-f5e4-4aac-b0f5-9c5fc63fcff2.png)